pkg/revocation/db_test.go

```diff
@@ -151,3 +151,38 @@ func TestRevocationDB_Put_error(t *testing.T) {
 		}
 	})
 }
+
+func TestRevocationDB_List(t *testing.T) {
+	ctx := testcontext.New(t)
+	defer ctx.Cleanup()
+
+	testrevocation.RunDBs(t, func(t *testing.T, revDB extensions.RevocationDB, db storage.KeyValueStore) {
+		keys, chain, err := testpeertls.NewCertChain(2, storj.LatestIDVersion().Number)
+		require.NoError(t, err)
+		keys2, chain2, err := testpeertls.NewCertChain(2, storj.LatestIDVersion().Number)
```

Comment:
I don't think you need a second chain. You can just revoke the leaf of the one chain multiple times.

Reply:
I tried using the same chain but it only produces 1 revocation. Is this correct or am I missing something?
`firstRevocation, err := extensions.NewRevocationExt(keys[peertls.CAIndex], chain[peertls.LeafIndex])`
`secondRevocation, err := extensions.NewRevocationExt(keys[peertls.CAIndex], chain[peertls.LeafIndex])`

Reply:
That actually makes perfect sense, my mistake. Here we're just creating a revocation extension, not actually modifying the chain by doing a full, proper, revocation. Also, I'm now realizing that revDB will only store the most recent revocation for the same nodeID. So even if we were modifying the chain you would still only have one revocation in the DB. what you did before was correct, sorry for the confusion.

Reply:
ok no worries, good to know, thanks!